### PR TITLE
Determine if column classes are needed.

### PR DESCRIPTION
**Pull request by SystemF**
Hi Dwight,

I made another addition when I found out that column styles of the index are always included even when they're not needed (e.g. for standard form style). They are only needed by Horizontal style forms I think.

Let me know!

### CODE REVIEW

**Comment by dwightwatson** [comment]
Hey, I really appreciate this addition but the code formatting is all off. I'll comment the things that need changing and if you're able to fix them up then I can bring the PR in.

File: src/BootstrapForm.php [hunk 1]
```diff
@@ -36,6 +36,13 @@ class BootstrapForm
      * @var \Illuminate\Session\Store
      */
     protected $session;
+	
+    /**
+     * Set to true for horizontal form.
+     *
+     * @bool
```
Comment (dwightwatson) [inline src/BootstrapForm.php]: Should be `@var bool`.

File: src/BootstrapForm.php [hunk 2]
```diff
@@ -36,6 +36,13 @@ class BootstrapForm
...
+     */
+    protected $use_column_classes = false;
```
Comment (dwightwatson) [inline src/BootstrapForm.php]: Variable names should be camel case. Perhaps a more descriptive variable name would be `$horizontalForm`?

File: src/BootstrapForm.php [hunk 3]
```diff
@@ -71,6 +78,7 @@ public function open(array $options = array())
 
             if ($defaultForm === 'horizontal') {
                 $options['class'] = 'form-horizontal';
+				$this->use_column_classes = true;
```
Comment (dwightwatson) [inline src/BootstrapForm.php]: Using tabs for indentation, and the indentation is all off.

File: src/BootstrapForm.php [hunk 4]
```diff
@@ -172,8 +181,12 @@ public function staticField($name, $label = null, $value = null, $options = arra
 
         $label = $this->getLabelTitle($label, $name);
         $inputElement = '<p'.$this->html->attributes($options).'>'.e($value).'</p>';
-
-        $wrapperOptions = ['class' => $this->getRightColumnClass()];
+		
+		$wrapperOptions = [];
+		if($this->use_column_classes){
```
Comment (dwightwatson) [inline src/BootstrapForm.php]: There should be a space between `if` and the opening bracket. There should also be a space between the closing bracket and the opening brace. Also, indentation is still off.

**Comment by SystemF** [comment]
Well, that's awkward, my IDE formatting settings where screwed. I now used the PSR-2 formatting standard in the way Laravel 5 was build (so even the namespace is on the same line as the PHP opening tag). I was used to write in CamelCase for Class and Function names but to use snake_case for variables. I've changed it to the suggested! Hope this will be better! ;)

**Comment by dwightwatson** [comment]
Awesome, thank you! I'll tag this release tonight.

**Pull request merged**